util/fips.py

```diff
@@ -0,0 +1,62 @@
+from smtplib import SMTPException
+from smtplib import SMTPAuthenticationError
+from smtplib import SMTPNotSupportedError
+
+# This method is the same as smtplib.SMTP.login except that CRAM_MD5 has been removed from preferred auths since MD5 is not FIPS compliant.
```

Comment:
For reference, https://github.com/mattupstate/flask-mail/blob/1709c70d839a7cc7b1f7eeb97333b71cd420fe32/flask_mail.py#L154 doesn't allow configuring the SMTP client other than the server and the port. So we can't do something `conn.esmtp_features['auth'] = 'PLAIN LOGIN'`